metadata/agg_json_data.py

```diff
@@ -142,7 +142,7 @@ def clean_df(df):
                     )
                 # Count the number of components calculated
                 elif meta_col == MetaConst.im_comp.value:
-                    df.loc[:, (sim_type, meta_col)] = [
+                    df.loc[:, (sim_type, MetaConst.im_comp_count.value)] = [
```

Comment:
I guess this was a bug?

Reply:
Yeah this was a bug. It meant that the component count was stored in the column that initially contained the component string (i.e. "['geom',..]") instead of in a separate column.